perfkitbenchmarker/analyze.py

```diff
@@ -96,107 +135,95 @@ def FullyInInterval(start_times, durations, interval_start, interval_duration):
     and false otherwise.
   """
 
-  interval_end = interval_start + interval_duration
-  record_ends = start_times + durations
+  assert start_times.index.equals(durations.index)
 
-  return (start_times >= interval_start) & (record_ends <= interval_end)
+  record_ends = start_times + durations
+  return (start_times >= interval.start) & (record_ends <= interval.end)
 
 
-def AllStreamsThroughputStats(durations, sizes, stream_ids,
-                              num_streams, interval_duration):
-  """Compute the net throughput of multiple streams doing operations.
+def ThroughputStats(start_times, durations, sizes, stream_ids, num_streams):
+  """Compute throughput stats of multiple streams doing operations.
 
   Args:
+    start_times: a pd.Series of POSIX timestamps, as floats.
     durations: a pd.Series of durations in seconds, as floats.
     sizes: a pd.Series of bytes.
     stream_ids: a pd.Series of any type.
     num_streams: int. The number of streams.
-    interval_duration: a float. The time all streams were active, in seconds.
 
-  durations, sizes, and stream_ids must have matching indices. This
-  function computes the per-stream net throughput (sum of bytes
+  start_times, durations, sizes, and stream_ids must have matching indices.
+  This function computes the per-stream net throughput (sum of bytes
   transferred / total transfer time) and then adds the results to find
   the overall net throughput. Operations from the same stream cannot
   overlap, but operations from different streams may overlap.
 
-  Returns: a tuple of
-    - The net throughput of all streams, excluding times they were inactive.
-    - The throughput of all streams, including times they were inactive.
-    - The total time that all streams were inactive.
-    - The total time all streams were inactive, as a proportion of the
-      total benchmark time.
+  Returns: a dictionary whose keys are metric names and whose values
+  are Quantity objects holding the metric values with appropriate
+  units.
```

Comment:
Expand this comment to include specifically what's returned - 

Returns: <your existing text> + "specifically, this includes:" 
 - blah